### PR TITLE
Change order of columns in spreadsheet

### LINKED ISSUE #3615: Deploy Lambda: change the order of the columns that are posted to the spreadsheet
We need to change the order of the columns in the spreadsheet. 
In order to do so we need to change the lambda code that pushes the data into the spreadsheet and put 'S3 Url' right after 'Run Name'

### PR DESCRIPTION
**Pull request by nikolayilyin**
Synced actual code with develop and formatted Simulate BEAM lambda.
Fixed calculation of RAM instance size.
Order of columns in the simulations spreadsheet is changed - S3 Url column is right after Simulation Name.

**Changes are already live**

- - -
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3628)

### CODE REVIEW

**Comment by nikolayilyin** [comment]
test!

File: aws/src/main/python/beam_lambda/lambda_function.py [hunk 1]
```diff
@@ -364,12 +364,14 @@
...
+                  'c5d.large', 'c5d.xlarge', 'c5d.2xlarge', 'c5d.4xlarge', 'c5d.9xlarge', 'c5d.18xlarge',
+                  'c5d.24xlarge',
                   'r5.large', 'r5.xlarge', 'r5.2xlarge', 'r5.4xlarge', 'r5.8xlarge', 'r5.12xlarge', 'r5.24xlarge',
-                  'r5d.large', 'r5d.xlarge', 'r5d.2xlarge', 'r5d.4xlarge', 'r5d.12xlarge', 'r5d.24xlarge',
+                  'r5d.large', 'r5d.xlarge', 'r5d.2xlarge', 'r5d.4xlarge', 'r5d.12xlarge', 'r5d.16xlarge',
+                  'r5d.24xlarge',
                   'm5d.large', 'm5d.xlarge', 'm5d.2xlarge', 'm5d.4xlarge', 'm5d.12xlarge', 'm5d.24xlarge',
                   'z1d.large', 'z1d.xlarge', 'z1d.2xlarge', 'z1d.3xlarge', 'z1d.6xlarge', 'z1d.12xlarge',
-                  'x2gd.metal', 'x2gd.16xlarge']
+                  'x2gd.metal', 'x2gd.16xlarge', 'x2gd.8xlarge', 'r5a.16xlarge', 'r5a.4xlarge']
```
Comment (dimaopen) [inline aws/src/main/python/beam_lambda/lambda_function.py]: Should this new instance type (`x2gd.8xlarge`) also be added to `instance_type_to_memory`?

File: build.gradle [hunk 2]
```diff
@@ -230,7 +234,8 @@ dependencies {
     testImplementation group: 'org.scalatest', name: "scalatest_${scalaBinaryVersion}", version: '3.2.9'
     testImplementation group: 'org.scalatestplus', name: 'scalatestplus-junit_2.12', version: '1.0.0-M2'
     testRuntimeOnly "org.pegdown:pegdown:1.6.0"
-    testImplementation group: 'com.vladsch.flexmark', name: 'flexmark-all', version: '0.36.8' // HTML report for scalatest
+    testImplementation group: 'com.vladsch.flexmark', name: 'flexmark-all', version: '0.36.8'
+    // HTML report for scalatest
```
Comment (dimaopen) [inline build.gradle:241]: Shouldn't the comment go before the line of code?

**Review by dimaopen — COMMENTED** [review]
2 small notes

File: build.gradle (hunk 2 above)
Comment (nikolayilyin) [reply build.gradle:241]: ok, the comment should definitely go before the code line

File: aws/src/main/python/beam_lambda/lambda_function.py (hunk 1 above)
Comment (nikolayilyin) [reply aws/src/main/python/beam_lambda/lambda_function.py]: That was in the lambda already, I just synced the code, but I do not see why we should avoid adding this instance.

**Comment by nikolayilyin** [comment]
test!

File: aws/src/main/python/beam_lambda/lambda_function.py (hunk 1 above)
Comment (nikolayilyin) [reply aws/src/main/python/beam_lambda/lambda_function.py]: Actually there are three instances that are not presented in instance_type_to_memory: 'x2gd.16xlarge', 'x2gd.8xlarge', 'x2gd.metal'

**Comment by nikolayilyin** [comment]
test!

**Review by dimaopen — APPROVED** [review]

**Pull request merged**